transport/src/main/java/io/scalecube/transport/TransportConfig.java

```diff
@@ -82,7 +91,20 @@ public Builder useNetworkEmulator(boolean useNetworkEmulator) {
       return this;
     }
 
+    public Builder withMessageCodec(MessageCodec messageCodec) {
```

Comment:
i suggest to rename method to messageCodec(MessageCodec messageCodec)
for me the with prefix was always confusing :)

Reply:
I apologize for not having noticed your code style